docs/conceptual/node/extrinsics.md

```diff
@@ -2,12 +2,91 @@
 title: Extrinsics
 ---
 
-* What is an Extrinsic?
+An extrinsic is a piece of information that comes from outside the chain and is included in a 
+block. Extrinsics fall into three categories: inherents, signed transactions, and unsigned 
+transactions.
 
-* What is a Signed Extrinsic?
+Note that [events](development/module/events.md) are not extrinsics. The chain emits events for 
+pieces of information that are intrinsic to the chain itself. For example, staking rewards are 
+events, not extrinsics, because the reward is triggered by circumstances intrinsic to the chain's 
+logic.
 
-* What is an Unsigned Extrinsic?
+## Block Structure
 
-* How do you validate an Inherent Extrinsic?
+A block in Substrate is composed of a header and an array of extrinsics. The header contains a  
+block height, parent hash, extrinsics root, state root, and digest. This section will only focus on 
+the extrinsics root.
 
-* What are signed extensions?
+Extrinsics are bundled together into a block as a series to be executed as each is defined in the 
+runtime. The extrinsics root is a cryptographic digest of this series. This serves two purposes. 
+First, it prevents any alterations to the series of extrinsics after the header has been built and 
+distributed. Second, it provides a means of allowing light clients to succinctly verify that any 
+given extrinsic did indeed exist in a block given only knowledge of the header.
+
+- [Block Reference](https://crates.parity.io/sp_runtime/traits/trait.Block.html)
```

Comment:
```suggestion
- [Block Reference](https://substrate.dev/rustdocs/master/sp_runtime/traits/trait.Block.html)
```

We've tried to link to rustdocs under the substrate.dev domain because we have versioning there (currently only v1.0 and master, but soon (hopefully) v2.0)